index.html

```diff
@@ -65,10 +65,29 @@ <h2>preferCurrentTab</h2>
         </dt>
         <dd>
           <p>
-            When {{MediaDevices/getDisplayMedia()}} is called, if <code>preferCurrentTab</code> is
-            <code>true</code>, the user agent SHOULD present the current tab as the most prominent
-            option. Otherwise, no such requirement is made.
+            When {{MediaDevices/getDisplayMedia()}} is called, the user agent MUST run the following
+            steps:
           </p>
+          <ol>
+            <li>
+              <p>Let <var>constraints</var> be the method's first argument.</p>
+            </li>
+            <li>
+              If <code>constraints.preferCurrentTab</code> is <code>false</code>, abort these steps.
+            </li>
+            <li>
+              If <code>constraints.</code>{{DisplayMediaStreamConstraints/selfBrowserSurface}} is
+              {{SelfCapturePreferenceEnum/"exclude"}}, return a promise <a>rejected</a>
+              with a {{DOMException}} object whose {{DOMException/name}} attribute has the value
+              {{InvalidStateError}}.
+            </li>
+            <li>
+              All other steps associated with {{MediaDevices/getDisplayMedia()}} should now be
+              applied. If these steps culminate in the user agent presenting the user with a
+              selection of display surfaces, the user agent SHOULD present the current tab as the
```

Comment:
Maybe link "display surfaces" to https://w3c.github.io/mediacapture-screen-share/#dfn-display-surface

Reply:
Done.